oat-sa/tao-item-runner-qti-fe
# feat: add figure element

Related to: https://oat-sa.atlassian.net/browse/AUT-120

**What was done:**

- added Figure and Figcaption model and renderer

For local test need to use brach from qti-sdk, please run  `composer require "qtism/qtism:dev-feature/legacy/TR-1499/support_qti_figure_figcaption_elements as 0.25.2" -W`

# How to test:

- create new passage
- add image to it
- add/edit image caption
- save
- check that it is rendered properly in preview, in items
